### DISCUSSION TITLE
Preparing the v7.2.3 release

### DISCUSSION THREAD

**Discussion by vmiklos** [vmiklos/plees-tracker#205] [General]
Hi,

I plan to release v7.2.3 towards the end of the next week. It currently contains new English-only strings.

@sguinetti @fabianski7 may I ask you to update the translations you already created in the past?

Test APK here: https://share.vmiklos.hu/plees-tracker/app-debug-v7.2.3-rc1.apk

Release notes: https://github.com/vmiklos/plees-tracker/blob/master/NEWS.adoc

Thanks,

Miklos

**Comment by vmiklos** [comment]
@sguinetti I already got an update from you, so please ignore this, sorry for the noise. :-)

**Comment by fabianski7** [comment]
Done. Sorry for the delay
